adafruit_platformdetect/board.py

```diff
@@ -30,7 +30,7 @@
 
 from adafruit_platformdetect.constants import boards, chips
 
-__version__ = "0.0.0+auto.0"
+__version__ = "3.72.1"
```

Comment:
Please change this back to 0.0.0+auto.0. The versioning is automatically changed this way.